test/compare_output_test.dart

```diff
@@ -145,7 +145,7 @@ void main() {
       final sep = '.'; // We don't care what the path separator character is
       final firstUnfoundExample = new RegExp('warning: lib${sep}example.dart: '
           '@example file not found.*test_package${sep}dog${sep}food.md');
-      if (!result.stderr.contains(firstUnfoundExample)) {
+      if (!result.stdout.contains(firstUnfoundExample)) {
```

Comment:
Do we know why this is now stdout?  Warning prints should go to stderr.

Reply:
Disagree! if dartdoc has a error/crash that should go to stderr

See https://www.jstorimer.com/blogs/workingwithcode/7766119-when-to-use-stderr-instead-of-stdout

stderr should be for diagnostics.

If I pipe the output from dartdoc to a file, I want the hints and the warnings all together.

Reply:
As discussed offline, this interpretation doesn't follow from the intended purpose of dartdoc, which is not to generate warnings, but documentation.  Given that, the warnings are diagnostic information, and should remain on stderr.

Reply:
See https://github.com/dart-lang/dartdoc/pull/1518/commits/d8e2a66e452e8425a181bee5d826e873c8c9e60e#r146899745